test/unit/world_location_test.rb

```diff
@@ -189,8 +177,7 @@ class WorldLocationTest < ActiveSupport::TestCase
 
   context "equality" do
     setup do
-      stub_worldwide_api_has_location("rohan")
-      stub_worldwide_api_has_location("gondor")
+      stub_worldwide_api_has_locations(%w[rohan gonder])
```

Comment:
Is `gonder` a typo?

Reply:
Yes .... and also an admission that I know nothing about Lord of the Rings!

Have fixed this.

Reply:
Same!